docs/api-reference/enrichment-components/configuration-reference/index.md

```diff
@@ -20,54 +20,24 @@ license {
-| `featureFlags.acceptInvalid` | Optional. Default: `false`. Enrich *3.0.0* introduces the validation of the enriched events against atomic schema before emitting. If set to `false`, a failed event will be emitted instead of the enriched event if validation fails. If set to `true`, invalid enriched events will be emitted, as before. |
+| `validation.acceptInvalid` | Optional. Default: `false`. Enrich *3.0.0* introduces the validation of the enriched events against atomic schema before emitting. If set to `false`, a failed event will be emitted instead of the enriched event if validation fails. If set to `true`, invalid enriched events will be emitted, as before. |
 | `validation.atomicFieldsLimits` (since *4.0.0*) | Optional. For the defaults, see [here](https://github.com/snowplow/enrich/blob/master/modules/common/src/main/resources/reference.conf). Configuration for custom maximum atomic fields (strings) length. It's a map-like structure with keys being atomic field names and values being their max allowed length. |
-
-Instead of a message queue, it's also possible to read collector payloads from files on disk. This can be used for instance for testing purposes. In this case the configuration needs to be as below.
-
-| parameter | description |
-|-|-|
-| `input.type`| Required. Must be `FileSystem`. |
-| `input.dir`| Required. E.g. `/input/collectorPayloads/`. Directory containing collector payloads encoded with Thrift. |
-
-Likewise, it's possible to write enriched events, pii events and failed events to files instead of PubSub or Kinesis.
-
-To write enriched events to files:
-
-| parameter | description |
-|------------------------|------------------------------------------------------------------------------------|
-| `output.good.type` | Required. Must be `FileSystem`. |
-| `output.good.file` | Required. E.g. `/output/enriched`. File where enriched events will be written. |
-| `output.good.maxBytes` | Optional. E.g. `1048576`. Maximum size of a file in bytes. Triggers file rotation. |
-
-To write failed events to files:
-
-| parameter | description |
-|-|-|
-| `output.bad.type` | Required. Must be `FileSystem`. |
-| `output.bad.file` | Required. E.g. `/output/bad`. File where failed events will be written. |
-| `output.bad.maxBytes` | Optional. E.g. `1048576`. Maximum size of a file in bytes. Triggers file rotation. |
-
-To write pii events to files:
-
-| parameter | description |
-|-|-|
-| `output.pii.type` | Required. Must be `FileSystem`. |
-| `output.pii.file` | Required. E.g. `/output/pii`. File where pii events will be written. |
-| `output.pii.maxBytes` | Optional. E.g. `1048576`. Maximum size of a file in bytes. Triggers file rotation. |
+| `validation.maxJsonDepth` | Optional. Default: `40`. Maximum allowed depth for the JSON entities in the events. Event will be sent to bad row stream if it contains JSON entity with a depth that exceeds this value. |
```

Comment:
```suggestion
| `validation.maxJsonDepth` (since *6.0.0*) | Optional. Default: `40`. Maximum allowed depth for the JSON entities in the events. Event will be sent to bad row stream if it contains JSON entity with a depth that exceeds this value. |
```